app/Models/Bible/BibleBook.php

```diff
@@ -22,14 +24,14 @@ class BibleBook extends Model
     protected $connection = 'dbp';
     protected $table = 'bible_books';
     public $incrementing = false;
-    public $fillable = ['abbr','book_id', 'name', 'name_short', 'chapters'];
+    public $fillable = ['abbr','book_id', 'name', 'name_short', 'chapters', 'bible_seq'];
```

Comment:
bible_seq should be book_seq everywhere in this file. there is no bible_seq

Reply:
@bradflood , It was my bad. So, I have updated the BibleBook model with the correct field name: `book_seq`